corehq/apps/registry/models.py

```diff
@@ -303,6 +306,16 @@ class Meta:
             ),
         ]
 
+    def to_json(self):
+        return {
+            "registry_slug": self.registry.slug,
+            "date": self.date.isoformat(),
```

Comment:
This'll be UTC, won't it? Can you use the user/domain timezone?

Reply:
5289e571708: Also updated the screenshot in the description